qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
+import re
+
+
+class KeysightB220X(VisaInstrument):
+    """
+    QCodes driver for B2200 / B2201 switch matrix
+    """
+
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name, address, terminator='\r', **kwargs)
+
+        self.add_function(name='clear_status',
```

Comment:
@WilliamHPNielsen What is the long term plan for functions?

Reply:
to deprecate them.

@StefanD986 , please, don't use functions, and create bound methods instead. Thanks!

Reply:
Good to know. I will change them to methods. (the add_function felt a bit cumbersome anyway)

Reply:
I changed all `add_function` to methods.